modules/core/src/main/scala/sangria/schema/Schema.scala

```diff
@@ -720,17 +721,22 @@ case class Argument[T](
     defaultValue: Option[(_, ToInput[_, _])],
     fromInput: FromInput[_],
     astDirectives: Vector[ast.Directive],
-    astNodes: Vector[ast.AstNode])
+    astNodes: Vector[ast.AstNode],
+    deprecationReason: Option[String])
```

Comment:
`Field` and `EnumValue` defines the fields in this order:
- `deprecationReason: Option[String]`
- `astDirectives: Vector[ast.Directive]`
- `astNodes: Vector[ast.AstNode]`

It'd be great to keep the same order for `InputField` and `Argument`.